src/lib/components/NglMoleculeViewer.react.js

```diff
@@ -21,9 +21,9 @@ export default class NglMoleculeViewer extends Component {
     }
 }
 
-const defaultHeight = '500px';
+const defaultHeight = '600px';
 
-const defaultWidth = '500px';
+const defaultWidth = '600px';
```

Comment:
Is it important to increase the default height and width? This has been released, so ideally we don't change its defaults, but if this is substantially better it's OK - we're still on 0.x and it hasn't been out long anyway.

Reply:
It's a minor change, but I think it's nice to have. The additional space by default gives a little bit more room to work with when moving the molecule around.